core-bundle/src/Image/ImageSizes.php

```diff
@@ -119,6 +139,15 @@ private function loadOptions(): void
 
         $options = [];
 
+        foreach ($this->predefinedSizes as $name => $imageSize) {
+            $options['image_sizes'][$name] = sprintf(
+                '%s (%sx%s)',
+                $this->translator->trans(substr($name, 1), [], 'image_sizes') ?: substr($name, 1),
+                $imageSize['width'],
```

Comment:
```suggestion
                $imageSize['width'] ?? '',
```

Reply:
@qzminski Why did you decide against @ausi's suggestion?

Reply:
@leofeyer I added it in wrong place 🙈see 8555e9796

Reply:
There are some more places where @ausi requested to change this. I have unresolved all matching conversations so you can track them more easily.

Reply:
Ah, no, my bad. You have already wrapped those places with `isset()`. Sorry.

Reply:
I decided to use `isset($val) ? (int) $val … : 0` instead of `(int) ($val ?? 0)` because – to be precise – there is no need to cast `0` to `int`.